doc/src/Howto_body.rst

```diff
@@ -270,6 +281,15 @@ A disk, whose diameter is 3.0, mass 1.0, is specified as follows:
    0 0 0
    3.0
 
+If the data file defines a general triclinic box, then the orientation
```

Comment:
This section (lines 284-292) duplicates the above (lines 164-172 for nparticle), same as lines 459-467 for body rounded/polyhedron. Three duplicated paragraphs are not specific to individual body styles.